server/repository/src/db_diesel/period/period_row.rs

```diff
@@ -60,4 +60,14 @@ impl<'a> PeriodRowRepository<'a> {
             .optional()?;
         Ok(result)
     }
+
+    pub fn find_many_by_program_schedule_ids(
```

Comment:
```suggestion
    pub fn find_many_by_period_schedule_ids(
```
No reference to program required here right?

Reply:
oops, thanks